views/components/icon-heading/_icon-heading.scss

```diff
@@ -0,0 +1,42 @@
+.icon-heading {
+  margin-top: 5rem;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  min-height: 2rem;
+  justify-content: flex-start;
+  .badge {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-basis: 4rem;
+    width: 4rem;
+    height: 4rem;
+    background-color: #D7FDFE;
+    border-radius: 50%;
+    margin: -0.75rem 0.25rem -0.75rem -1rem;
```

Comment:
I looks to me (not an authority) like we could do with a little more spacing on the right here.

Reply:
You're probably right. 🤔 I'll play around with the right spacing a few more times.